### DISCUSSION TITLE
firmware help for ender 3 v2 with motherboard 4.2.7 with BLtouch

### DISCUSSION THREAD

**Discussion by Failure2Society** [Jyers/Marlin#1472] [Q&A] [answered]
I am having issues with getting the bltouch to work properly after adjusting settings to fit my ender 3 v2 printer, I am adjusting bed size, along with enabling bltouch functions, and adjusting some of the fine adjustment settings to fit my ender 3 v2. everything I try outside of those settings doesn't work for making the bltouch system work, I can back track to my old firmware to get it working, but the new firmware doesn't work for me. any help would be greatly appreciated.

**Comment by Jyers** [comment, marked as answer]
Make sure you're starting with the templates provided in the configuration files folder. Also noting specifically what settings are not working would help to better diagnose the problem
